modules/server/src/network/gql.ts

```diff
@@ -1,13 +1,26 @@
 import axios from 'axios';
-import { print } from 'graphql';
+import { ASTNode, print } from 'graphql';
 
-export const createGqlQuery = ({ query: queryAST, variables }) => {
+/**
+ * Creates a graphql query string with variables for use in a POST request
+ * @returns string
+ */
+export const createGqlQuery = ({
+	query: queryAST,
+	variables,
+}: {
+	query: ASTNode;
+	/**
+	 * TODO: Add typing for variables. Type will be dependant on gql endpoint being queried.
+	 */
+	variables: unknown;
```

Comment:
Looking at how this is used, we don't really care about the specific contents of the variables. This function is doing no work to ensure that the variables expected in the `query` are included in `variables` (or vice versa). The only thing we know is we want to be able to run `JSON.stringify(variables)` which takes `any`. In practice, we expect this parameter to be an object with some variable values which themselves could be anything, and we probably don't want it to be `string` or `number` etc.  So in this case I might reccomend:

```ts
variables: Record<string, any>
```